pkg/controller/machinepool/ibmcloudactuator_test.go

```diff
@@ -195,6 +195,6 @@ func generateIBMCloudMachineSetName(leaseChar, zone string) string {
 	return fmt.Sprintf("%s-%s-%s", testInfraID, leaseChar, zone)
 }
 
-func mockGetVPCZonesForRegion(ibmClient *mockibm.MockAPI, zones []string, region string) {
+func mockGetVPCZonesForRegion(ibmClient *mockibm.MockAPI, zones []string) {
```

Comment:
A true mock for [GetVPCZonesForRegion](https://github.com/openshift/hive/blob/2e3237f6cc7384090cc4ddef5054b259805fadaa/pkg/ibmclient/client.go#L393) should be using the arg `region` instead of hardcoded `testRegion` but given that we always called it with `testRegion` anyways, I guess this is alright.